<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsrInterface</class>
 <widget class="QWidget" name="UsrInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_usrpicture">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>picture</string>
   </property>
  </widget>
  <widget class="QTreeView" name="tv_relation">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>271</width>
     <height>701</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_search">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>780</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
